costmodel/costmodel.go

```diff
@@ -1378,6 +1378,12 @@ func (cm *CostModel) costDataRange(cli prometheusClient.Client, clientset kubern
 	}
 	clusterID := os.Getenv(clusterIDKey)
 
+	durHrs := end.Sub(start).Hours() + 1
+	durStr := fmt.Sprintf("%fh", durHrs)
+	if durHrs > 24.0 {
```

Comment:
it's ok to drop this and just print hours, we can convert. less code is better here esp since fmt.Sprintf(...) can have errors and we're trying to turn this around quickly.